### PR TITLE
Update ticker to show number of supporters

### PR DESCRIPTION
**Pull request by tomrf1**
With this change, the ticker can operate in one of 2 modes:
- money - fetches data from `/ticker.json` and displays with currency symbol
- people - fetches data from `/supporters-ticker.json`

I've also made the ticker component more configurable, so that copy can be passed in as props.

I also refactored the ticker component a bit because it was a weird mix of component state and just variables on the class. I consolidated everything into the component's state type.

## Before goal:
Screen Shot 2020-06-04 at 13 52 57

## After goal:
Screen Shot 2020-06-04 at 13 51 39

### CODE REVIEW

File: support-frontend/assets/helpers/campaigns.jsx [hunk 1]
```diff
@@ -1,15 +1,7 @@
 // @flow
 
 import type { ContributionTypes } from 'helpers/contributions';
-
-export type TickerType = 'unlimited' | 'hardstop';
-
-export type TickerSettings = {
```
Comment (tomrf1) [inline support-frontend/assets/helpers/campaigns.jsx:7]: moved to `contributionTicker.jsx`

**Comment by ionamckendrick** [comment]
The padding between the two lines on the second ticker is a bit tight... could we possibly increase this by a few px? @tomrf1

**Comment by tomrf1** [comment]
@ionamckendrick done!

**Comment by JoemG** [comment]
Looks great 💯

**Pull request merged**